screens/Settings/BackupToggle.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {useState} from 'react';
+import {Switch} from 'react-native-elements';
+import {Text} from '../../components/ui';
+import {Modal} from '../../components/ui/Modal';
+import {Theme} from '../../components/ui/styleUtils';
+import {useBackupScreen} from './BackupController';
+import {Platform} from 'react-native';
+import {MessageOverlay} from '../../components/MessageOverlay';
+
+export const BackupToggle: React.FC<BackupToggleProps> = props => {
+  const [dataBackup, setDataBackup] = useState(false);
+
+  const controller = useBackupScreen(props);
+
+  // TODO : Check if the setup is already done
```

Comment:
What is this ToDo about?

Reply:
This to-do was added when we set up the password and phone number for cloud backup. As this story is for local backup creation, This TODO can be removed.